sources/engine/Stride.Engine/Profiling/DebugTextSystem.cs

```diff
@@ -90,16 +120,15 @@ public override void Draw(GameTime gameTime)
                 };
             }
 
-            if (overlayMessages.Count == 0)
-                return;
-
```

Comment:
I think you can still keep this, if you don't have any message scheduled you don't need the rest of your logic to run ?

Reply:
Never mind me, this is fine !